src/backend/gl/gl_common.c

```diff
@@ -800,6 +805,9 @@ uint64_t gl_get_shader_attributes(backend_t *backend_data attr_unused, void *sha
 }
 
 bool gl_init(struct gl_data *gd, session_t *ps) {
+	glGenQueries(2, gd->frame_timing);
```

Comment:
i don't see a matching `glDeleteQueries` call. i suppose it should be in the `gl_deinit` function.